app/webpacker/components/Panel/pages/RunValidatorsPage/ValidationOutput.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Header, Message } from 'semantic-ui-react';
+import Loading from '../../../Requests/Loading';
+import Errored from '../../../Requests/Errored';
+import ValidationListView from './ValidationListView';
+
+export default function ValidationOutput({
+  validationOutput, isFetching, isError, showCompetitionNameOnOutput,
+}) {
+  if (isFetching) return <Loading />;
+  if (isError) return <Errored />;
+
+  if (!isFetching && !isError && !validationOutput) {
```

Comment:
At this point, you already know that `!isFetching && !isError` will always be true. Otherwise, the code would have returned already. So just checking `!validationOutput` should be enough.

Reply:
Oh yes, I missed it for some reason. Changed it.